references/synonyms.md

```diff
@@ -1,17 +1,17 @@
 # Synonyms
 
-## List one synonym
+## List synonyms of one sequence
```

Comment:
Should sequence become a word in our terminology? I used it in the documentation because I liked it, but maybe `string` could be better.

Reply:
I read it because I needed to understand synonyms and I did not understand 😂 That's why I changed. For me, we don't *list* one thing. Really confusing in a title for me.
But we have to discuss because I'm not sure at all what is a sequence, even after reading the docs.